analyses/independent-samples/00-repeated-samples.Rmd

```diff
@@ -32,21 +32,21 @@ set.seed(201910)
 ```{r load files}
 root_dir <- rprojroot::find_root(rprojroot::has_dir(".git"))
 if(params$base_run == 0) {
-sample_df <- readr::read_tsv(file.path(root_dir, "data", "pbta-histologies.tsv"), guess_max = 10000)
+sample_df <- readr::read_tsv(file.path(root_dir, "data", "histologies.tsv"), guess_max = 10000)
```

Comment:
```suggestion
sample_df <- readr::read_tsv(file.path(root_dir, "data", "histologies.tsv"), guess_max = 10000)
```

Should we add a higher number in `guess_max` @kgaonkar6 since this df is going to be very large (over 10k now)?

Reply:
Yeah we should increase the value here that will remove the warnings for the `gtex_group` column that get's read incorrectly.